frontend/app/src/components/home/ChatMessageMenu.svelte

```diff
@@ -272,11 +278,50 @@
                 </div>
             </HoverIcon>
         {/each}
+        {#if canReact && !failed}
+            <HoverIcon compact={true} onclick={() => showEmojiPicker()} title={$_("pickEmoji")}>
```

Comment:
Instead of `compact={true}` you can just use the shorthand `compact`

A few more similar cases below.